source/_integrations/knocki.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: Knocki
+description: Instructions on how to setup Knocki devices in Home Assistant.
+ha_category:
+  - Event
+ha_config_flow: true
+ha_release: 2024.7
+ha_iot_class: Cloud Push
+ha_codeowners:
+  - '@jgatto'
+  - '@joostlek'
+ha_domain: knocki
+ha_platforms:
+  - event
+ha_integration_type: integration
+---
+
+The **Knocki** {% term integration %} can receive events for the triggers set up from your [Knocki devices](https://knocki.com/).
+
```

Comment:
Are there any prerequisites? For example: Do I need a Knocki account and login credentials when adding the integration?